packages/waku/src/lib/middleware/rsc/ssr.ts

```diff
@@ -340,9 +318,11 @@ export const renderHtml = async <Context>(
                     filePath.slice(wakuDist.length).replace(/\.\w+$/, '');
                   return { id, chunks: [id], name };
                 }
-                const id = url
-                  .pathToFileURL(transpile!(filePath, name))
-                  .toString();
+                const id =
+                  url
+                    .pathToFileURL(filePath)
+                    .toString()
+                    .slice('file://'.length) + '#dev';
```

Comment:
I wasn't sure if this was right.
As CI passes, I'll merge it. Feel free to follow up if needed.